src/main/java/ca/bc/gov/ag/efax/mail/service/EmailServiceImpl.java

```diff
@@ -117,7 +117,12 @@ private boolean processMessage(MailMessage mailMessage) throws Exception {
                 }
             }
             
-            message.sendAndSaveCopy(WellKnownFolderName.SentItems);
+            if (exchangeProperties.getSaveInSent()) {
```

Comment:
Was this meant to be commited on this PR?

Reply:
It could be different PR. I noticed the env wasn't used when it should have been.